src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -156109,5 +156109,161 @@ void F(object? o)
                 Diagnostic(ErrorCode.ERR_ImplicitlyTypedArrayNoBestType, "stackalloc[] { x => x }").WithLocation(9, 18)
                 );
         }
+
+        [Fact, WorkItem(61361, "https://github.com/dotnet/roslyn/issues/61361")]
+        public void NullableBoolInVariousTests()
```

Comment:
It wasn't clear to me that this test relates to the bug being fixed.

Reply:
Initially, I was mistaken and thought that the lifted operator returned a `bool?`. But for built-in relational operations it returns a `bool`. Still, I left this test related to `bool?`.